extensions/vscode/package.json

```diff
@@ -325,6 +330,10 @@
           "command": "posit.publisher.homeView.showSelectConfigForDeployment",
           "when": "webviewId == 'posit.publisher.homeView' && webviewSection == 'even-easier-deploy-more-menu-matching-configs'"
         },
+        {
+          "command": "posit.publisher.homeView.showAssociateDeployment",
+          "when": "webviewId == 'posit.publisher.homeView' && webviewSection == 'even-easier-deploy-more-menu-matching-configs' && posit.publish.selection.haveCredentialMatch == 'true'"
```

Comment:
This shows the context menu option while a pre-deployment shows the association link.

We could try to hide it using message passing or we could generalize the label to remove the "with a **Different** Deployment". 🤔

Reply:
I'm not following you. Can you provide a screenshot? I'm looking at
![image](https://github.com/user-attachments/assets/8afb6d76-5f82-4901-93a6-ee6f0e7af6f9)

Reply:
Yeah that screenshot shows what I'm talking about. We have both "update that previous deployment" available as a link, and the "Associate with a Different Deployment on Connect". The wording is just strange since it isn't associated with any deployment yet. That may be too in the weeds with how this is working though.

Reply:
I've added another context state within the extension, which indicates if the actively selected content record is a PreContentRecord or not. This allows the menu item to now be shown only when it has been deployed, which should address the confusion you identified here.

Reply:
Thanks for doing that, it is a small thing, but it should make it easier to understand for users.